snuba/datasets/storages/metrics.py

```diff
@@ -76,7 +76,12 @@
     ),
     query_processors=[],
     stream_loader=build_kafka_stream_loader_from_settings(
-        processor=CounterMetricsProcessor(), default_topic=Topic.METRICS,
+        processor=CounterMetricsProcessor(),
```

Comment:
@ahmedetefy just to double check, you said that making this conditional as with sessions is not necessary, right?

https://github.com/getsentry/snuba/blob/b55fe5027878533f065da8d4b3025cb55c8607eb/snuba/datasets/storages/sessions.py#L120-L132

Reply:
Yes since it is not in production yet according to @fpacifici